hw/ip/entropy_src/doc/_index.md

```diff
@@ -10,83 +10,145 @@ This module conforms to the [Comportable guideline for peripheral functionality.
+
+In addition to the brief, low-latency health checks, various long-term statistics are accumulated in registers for additional diagnostic purposes or for in-depth analysis.
 For the hardware interface, entropy bits will be transferred whenever the downstream consumer is ready to receive them.
 
+At a high-level, this ENTROPY_SRC block, when enabled, will continuously collect entropy bits from the entropy source into a FIFO that can be read from the TL-UL bus, or sent out through a hardware interface.
+The PTRNG noise source will generate a relatively slow set of bit streams, with a total bandwidth on the order of 100 kbps.
+The digital source can be programmed to be faster, or much slower if desired.
+A digital source rate of flexible speed might be useful in demonstrating how the downstream applications use the entropy.
+
+Once a noise source has been selected through configuration, the raw noise data processing begins.
+This ENTROPY_SRC block will do health checks, collect and pack bits, and finally send them through a conditioning unit.
+There are two modes in which entropy bits are delivered, boot-time and FIPS/CC mode.
+Boot-time mode will deliver bits sooner for specific on-boot obfuscation applications, though the bits may not yet have been subjected to the same level of startup health checks required for FIPS or CC compliance.
+
+In boot-time mode health checks only operate on a window of 384 bits.
+The boot-time health checks are the same as the FIPS/CC health-checks, though with different thresholds.
+They are sensitive to the same types of statistical defects, though at reduced statistical resolution.
+With suitable thresholds, the boot-time health checks can be operate both with low false-alarm rates (low &alpha;), while still confirming with low &beta; that the total entropy of the first seed contains at least 80 bits of total entropy.
+During start up the initial 384 bits are held in a buffer until the boot-time start-up health checks are performed.
+Storing the seed in this buffer, allows this seed to released to the CSRNG immediately after the entropy has been confirmed.
+
+Boot-time mode also has the feature that it bypasses the SHA conditioning block, as only 384 bits are used in the initial boot-time seed.
+
+For maximal flexibility in normal operation, the conditioning function can also be implemented by firmware.
+When this firmware conditioning feature is activated, data read directly out of the noise source can be reinjected into the entropy pipeline via a TL-UL register after it has been processed by firmware.
+It should be noted that this firmware algorithm must be vetted by NIST to satisfy the requirements for a full-entropy source.
+This feature can also be disabled for security purposes, either by locking the feature via the {{< regref "REGEN" >}} register at boot, or by a write to one-time programmable (OTP) memory.
```

Comment:
this disable bit probably needs to be part of the design attestable state, if it's not already